### ISSUE TITLE
Videos: a case for support

### ISSUE THREAD

**Issue by kristinashu** [MozillaFoundation/Design#641]
From Jessie K

> We are producing a "case for support" video that features our Fellows and their stories in an intimate way. We will use this video alongside our solicitation/stewardship comms, and it will serve as evergreen donor comms that we expect to use in a variety of ways.

> We are working with MoCo on production, and we hope to use some of the graphic elements Natalie created for the last video. 

> Filling in this brief to flag:
> - an ask for some design team time for graphics for the video (likely Natalie's templated stuff)
> - support thinking through distribution channels for the video
> - heads up on fundraising campaigns that may use this video and implicate engagement (e.g. putting the video on a landing page)

> [Doc](https://docs.google.com/document/d/1sodOzThF9rLSOKTXjLzdNxB8ISHl1iC_dR5ngvk11g0/edit?ts=5f34717f)

tl;dr videos will need bumpers

**Comment by anilkanji** [comment]
Rainer needs Title slides and lower thirds - will send the edit to @natalieworth  mid w/o Sept 28

**Comment by natalieworth** [comment]
I spoke with Rainer and we have created a short test with [the type/ graphics added](https://drive.google.com/drive/u/1/folders/1OPC6vF1wOxkC-r2qOVswB8mnXp_twjri). 

There was a desire from the fellow/fundraising team to make the video feel less 'talking-heads' and to provide some more context to what each of the fellows does. Rainer suggested showcasing some of the regions of the fellows along with information about what they are working on.

**Comment by natalieworth** [comment]
@jamilasnell @phildexter I got the video yesterday from Rainer. I'm waiting on feedback from the Fundraising/ Fellow team for the draft version of the graphic treatment. Is it okay if we move this to the next sprint as time-wise I won't be able to get it done + reviews in a day and a half (and I'm off next week until Thursday)?

**Comment by kristinashu** [comment]
Yes! Move this to next sprint, can you please just let Jesse W know?

**Comment by natalieworth** [comment]
Spoke with Jessie and Rainer today. Rainer is going to send me the video files by tomorrow EOD and I have asked Jessie's team for the names, titles, locations, and content to include for the graphics by Wednesday.

_Almost unblocked_ :)

**Comment by natalieworth** [comment]
Have the file from Rainer and spoke with Jessie (she's hoping to have content to me asap).
